test/integration/visitor_can_view_deal_page_test.rb

```diff
@@ -8,7 +8,7 @@ class VisitorCanViewDealPageTest < ActionDispatch::IntegrationTest
     visit '/deal'
 
     assert page.has_content?("Today's Gnarliest Item!")
-    assert page.has_content?('gnar possum')
+    assert page.has_content?('Gnar possum')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.